tests/unittests/Foundation/NSRegularExpressionTests.m

```diff
@@ -174,6 +174,11 @@ NSUInteger checkMatches(NSRegularExpression* regex, NSString* testString) {
 
     ASSERT_EQ(theRange.location, NSNotFound);
     ASSERT_EQ(theRange.length, 0);
+
+    StrongId<NSTextCheckingResult> result =
```

Comment:
The rest of the code in the file does exactly that.  Went with when in Rome rule here :(.  

---
In reply to: [95231289](https://github.com/Microsoft/WinObjC/pull/1657#discussion_r95231289) [](ancestors = 95231289)